protocols/grpc/hyperstart.proto

```diff
@@ -103,6 +103,7 @@ message CreateSandboxRequest {
 	string hostname = 1;
 	repeated string dns = 2;
 	repeated Storage storages = 3;
+	bool shared_pidns = 4;
```

Comment:
It seams that all the containers are forced to be **shared** the sandbox pidns.

I prefer create_sandbox_pidns or sandbox_pidns or infra_pidns.

Reply:
Done.